fixtures/u2_type_alias.go

```diff
@@ -0,0 +1,4 @@
+package fixtures
+
+type Testtype1 struct {}
+type Testtype2 Testtype1
```

Comment:
It's more like typedef. Type alias is `type T1 = T2` (Go 1.9).

Reply:
That's why it failed for me in Go 1.8 :) updating to 1.10... anyway I've updated the spec to add a Typedef too, since many languages support both concepts.